### DISCUSSION TITLE
Dump require GLIBC

### DISCUSSION THREAD

**Discussion by simexg** [dbeaver/dbeaver#15928] [General]
#### System information: 
- Operating system (distribution) and version: Fedora 35
- DBeaver version: 21.3.3.202201221033
- Additional extensions: I didn't install any extensions.

#### Connection specification:
- Database name and version:  Mariadb 10.5.13
- Driver name: mariadb-java-client-2.7.1
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No

#### Describe the problem you're observing:
I tried to dump tables. Then: "Task execution failed. Error executing process
  Process failed (exit code = 1). See error log."

#### Include any warning/errors/backtraces from the logs
/run/user/1000/doc/c1194890/bin/mysqldump --add-drop-table --disable-keys --extended-insert --comments -u mysqluser --host=localhost --port=3306 bb_extra_tables types reason account_properties resolve_modes categories bookingstates processes states account_types
Task 'MySQL dump' started at Wed Jan 26 13:42:44 CET 2022
/run/user/1000/doc/c1194890/bin/mysqldump: /usr/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /run/user/1000/doc/c1194890/bin/mysqldump)

**Comment by SubOptimal** [comment]
It seems your `mysqldump` is not the version provided by Fedora 35.

When setting up the local client, specify the directory `/usr/bin` as the client home directory.

![image](https://user-images.githubusercontent.com/653288/151721746-cd0dc707-0d75-4101-951e-14026e982aa3.png)

Then dumping a table will use that executable.

![image](https://user-images.githubusercontent.com/653288/151721791-705bb56b-a13b-42a9-904b-accdbc32ba99.png)

**Comment by simexg** [comment]
Sorry it's a Flatpak install. When i select the /usr/bin folder the path in the information fields changes to /run/user/1000/doc/c1194890/bin. And I can't edit the path. I have only the brows window. So I can't force it to use /usr/bin path.

**Comment by SubOptimal** [comment]
For Flatpak installed DBeaver, there were some solutions discussed on #5008. Possibly this could solve your problem.
